pkg/controller/master/setting/harvester_rancher_sync.go

```diff
@@ -0,0 +1,59 @@
+package setting
+
+import (
+	"reflect"
+
+	harvesterv1 "github.com/harvester/harvester/pkg/apis/harvesterhci.io/v1beta1"
+	"github.com/harvester/harvester/pkg/settings"
+)
+
+const (
+	rancherUIDashboardIndexSettingName   = "ui-dashboard-index"
+	rancherUIOfflinePreferredSettingName = "ui-offline-preferred"
```

Comment:
can we use ranchersettings.UIDashboardIndex.Name and ranchersettings.UIOfflinePreferred.Name ?
```go
import (
	ranchersettings "github.com/rancher/rancher/pkg/settings"
)

```